core/store/package.rs

```diff
@@ -25,4 +29,27 @@ impl Package {
     pub fn deps(&self) -> &[Archive] {
         self.deps.as_ref()
     }
+
+    pub async fn update(&self, target: Target) -> Result<(), FromFileError> {
+        let path = self.get_local_manifest_path(&target);
+        let local_manifest = PackageManifest::from_file(&path).await.unwrap_or_default();
+
+        let new_manifest = self.manifest();
+        let merged = local_manifest.merge(new_manifest);
+
+        merged.write(&path).await
+    }
+
+    fn get_local_manifest_path(&self, target: &Target) -> PathBuf {
```

Comment:
Should we blow up here if we don't receive a Target::Fs ?

Reply:
Well, no, at this stage if we have a Package that means we packaged something correctly. If its an FsTarget then we have a path, that's great. But we can also package and upload RemoteTargets.